From 5f1cc1fe8b705c87126a5694fc5189596282987c Mon Sep 17 00:00:00 2001
From: FrancescoFerraro <francesco.f@variscite.com>
Date: Mon, 31 Oct 2022 17:36:26 +0100
Subject: [PATCH 07/12] imx8mm_var_dart: disable EFI flags

This is need to avoid an issue related to ARM System Ready tests.
Testing Capsule Update the following error happens:
CapsuleApp: failed to update capsule - Security Violation.

Signed-off-by: FrancescoFerraro <francesco.f@variscite.com>
(cherry picked from commit d6ab54e237dd9751ac09a5b0f2702a5b73d12abe)
Signed-off-by: Vanessa Maegima <vanessa.maegima@foundries.io>
---
 configs/imx8mm_var_dart_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configs/imx8mm_var_dart_defconfig b/configs/imx8mm_var_dart_defconfig
index dfe5196b6b6..ba3a69ae569 100644
--- a/configs/imx8mm_var_dart_defconfig
+++ b/configs/imx8mm_var_dart_defconfig
@@ -152,5 +152,4 @@ CONFIG_EFI_RUNTIME_UPDATE_CAPSULE=y
 CONFIG_EFI_CAPSULE_ON_DISK=y
 CONFIG_EFI_IGNORE_OSINDICATIONS=y
 CONFIG_EFI_CAPSULE_FIRMWARE_RAW=y
-CONFIG_EFI_CAPSULE_AUTHENTICATE=y
 CONFIG_EFI_SECURE_BOOT=y
-- 
2.34.1

